data/data.json

```diff
@@ -5383,5 +5339,5 @@
     }
   ],
   "errors": [],
-  "timestamp": 1549485129335
+  "timestamp": 1549907642205
```

Comment:
huh, at some point we should probably save the timestamp as a human readable string